Source/WebKit/WebProcess/WebPage/gtk/WebPrintOperationGtk.h

```diff
@@ -119,6 +129,10 @@ class WebPrintOperationGtk {
 
 #if USE(CAIRO)
     RefPtr<cairo_t> m_cairoContext;
+#elif USE(SKIA)
+    SkDynamicMemoryWStream m_skiaBuffer;
+    sk_sp<SkDocument> m_skiaDocument;
+    SkCanvas* m_pageCanvas { nullptr };
```

Comment:
I wonder if there's a more semantic way to store these fields. Perhaps with an auxiliary structure or class. Thoughts?